cspell.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
adding a lot of files to the root dir, can we stash them all somewhere, like `scripts/cspell/*`?

Reply:
Made the fixes